pyproject.toml

```diff
@@ -1,35 +1,65 @@
 [build-system]
 
 requires = ["setuptools",
-            "setuptools_scm",
+            "setuptools_scm[toml]",
             "wheel"]
 
 build-backend = 'setuptools.build_meta'
 
+[ tool.gilesbot ]
+#  [ tool.gilesbot.circleci_artifacts ]
+#    enabled = true
+#
+#  [ tool.gilesbot.circleci_artifacts.figure_report ]
+#    url = ".tmp/py38-figure/figure_test_images/fig_comparison.html"
+#    message = "Click details to see the figure test comparisons, for py38-figure."
+#    report_on_fail = true
+#
+#  [ tool.gilesbot.circleci_artifacts.figure_report_devdeps ]
+#    url = ".tmp/py39-figure-devdeps/figure_test_images/fig_comparison.html"
+#    message = "Click details to see the figure test comparisons for py39-figure-devdeps."
+#    report_on_fail = true
+
+  [ tool.gilesbot.pull_requests ]
+    enabled = true
+
+  [ tool.gilesbot.towncrier_changelog ]
+    enabled = true
+    verify_pr_number = true
+    changelog_skip_label = "No Changelog Entry Needed"
+    help_url = "https://github.com/sunpy/sunxspex/blob/main/changelog/README.rst"
+
+    changelog_missing_long = "There isn't a changelog file in this pull request. Please add a changelog file to the `changelog/` directory following the instructions in the changelog [README](https://github.com/sunpy/sunpy/blob/main/changelog/README.rst)."
+
+    type_incorrect_long = "The changelog file you added is not one of the allowed types. Please use one of the types described in the changelog [README](https://github.com/sunpy/sunpy/blob/main/changelog/README.rst)"
```

Comment:
The `sunxspex` default branch is still `master` (that can probably be changed?) so the `help_url` is currently 404. The updated towncrier types here seem to correspond to the `sunpy/sunpy` version which is out of sync with the `sunpy/sunxspex` changelog readme (`changelog/README.rst`). Maybe this readme should also be re-synced with the sunpy version, and the URLs in `changelog_missing_long` and `type_incorrect_long` above updated to point to the README within the `sunxspex` repo.